src/apps/tco2_dashboard/app.py

```diff
@@ -111,16 +114,28 @@ def get_data_pool():
     return df_deposited, df_redeemed
 
 
+def get_verra_data():
+    r = requests.post(
+        'https://registry.verra.org/uiapi/asset/asset/search?$maxResults=2000&$count=true&$skip=0&format=csv',
```

Comment:
We may need to bump this `maxResults=2000` at some point to avoid cutting off the data